bolt/runtime/common.h

```diff
@@ -151,10 +151,12 @@ struct timespec {
   uint64_t tv_nsec; /* nanoseconds */
 };
 
-#if defined(__aarch64__)
+#if defined(__aarch64__) || defined(__arm64__)
 #include "sys_aarch64.h"
-#else
+#elif defined(__x86_64__)
 #include "sys_x86_64.h"
+#else
+ exit(1);
```

Comment:
it is being used before like this, I have decided to avoid touch as didn't know clear intentions to fail in this case. 
But make sense to make it as '#error', maybe then in another place as well